riverdriver/riverpgxv5/river_pgx_v5_driver.go

```diff
@@ -99,6 +99,20 @@ func (e *Executor) JobCountByState(ctx context.Context, state rivertype.JobState
 	return int(numJobs), nil
 }
 
+func (e *Executor) JobDelete(ctx context.Context, id int64) (bool, error) {
+	job, err := e.queries.JobDelete(ctx, e.dbtx, id)
+	if err != nil {
+		return false, interpretError(err)
+	}
+	if job == nil {
+		return false, rivertype.ErrNotFound
```

Comment:
Doesn't sqlc always return a not found error for a `:one` query rather than return a `nil`?

What do you think about adding a driver test case that verifies that a `JobDelete` on a non-existent row just to make sure the right thing happens here?

Reply:
Added! Thought I already had that one but obviously missed it. And you're correct, this clause is unnecessary.